app/templates/meteor/_packages

```diff
@@ -4,10 +4,9 @@
 # but you can also edit it by hand.
 
 standard-app-packages
-coffeescript<% if (cssPreprocessor == 'stylus') { %>
-stylus<% } %><% if (cssPreprocessor == 'less') { %>
-less<% } %><% if (cssPreprocessor == 'bootstrap') { %>
-bootstrap<% } %>
+coffeescript<% if (cssPreprocessor == 'nemo64:bootstrap') { %>
```

Comment:
Maybe I'm wrong, but won't the `less` package automatically be included in the project if we use one of the bootstrap packages (since they include Less as a dependency)?